Marlin/Configuration.h

```diff
@@ -2249,6 +2249,9 @@
   #define Z_SAFE_HOMING_X_POINT X_CENTER  // X point for Z homing
   #define Z_SAFE_HOMING_Y_POINT Y_CENTER  // Y point for Z homing
   //#define Z_SAFE_HOMING_POINT_ABSOLUTE  // Ignore home offsets (M206) for Z homing position
+  #define Move_to_Custom //move nozzle to x y after a home
+  #define Custom_Pos_X 0 //or pick custom
+  #define Custom_Pos_Y 0	
```

Comment:
All of these defines should be capitalized and disabled by default. `CUSTOM_POS_X` / `CUSTOM_POS_Y` should only be enabled if `MOVE_TO_CUSTOM` is enabled. These defines should also have `_POINT` added to them / be renamed for clarity:

```cpp
  //#define MOVE_TO_CUSTOM_XY_POINT // Move nozzle to custom X/Y position after homing Z
  #if ENABLED(MOVE_TO_CUSTOM_XY_POINT)
    //#define CUSTOM_POS_X_POINT 0
    //#define CUSTOM_POS_Y_POINT 0
  #endif
```

... or simplify things and add some kind of post `Z_SAFE_HOMING` option like:

```cpp
  //#define Z_SAFE_HOMING_COMMANDS_POST "G0 X0 Y0" // GCODE commands that will be executed after homing Z
```

Reply:
Fixed the issues, I do agree the command post would be a good option as well, I did type out in the description but for some reason it deleted it, it would be good to move to true home after every home vs just with safe home, sometimes pritners have MAX end stops and the nozzle parks at them vs going to HOME, and this confuses some new users into thinking that HOME is where ever the end stop is vs the true home of front left 0,0

Reply:
> sometimes pritners have MAX end stops and the nozzle parks at them vs going to HOME, and this confuses some new users into thinking that HOME is where ever the end stop is vs the true home of front left 0,0 

No, that is correct.

Home is where your endstops are, which isn’t always X0/Y0. Home could be X-15/Y415, X310/Y-10, or any other combination of Min/Max values.

Reply:
> An ender 3 and ender5 for example are essentially the same machine, but the X axis on the 3 has a MIN X end stop and the ender 5 has a MAX X end stop. you can print the same file on either machine (some limitations) Each machine will auto home the same way, and print the file in the same orientation.

Sort of. A stock Ender-5 will print an Ender-3 G-code 180° out from an Ender-3 since Creality ships an Ender-5 with Ender-3 firmware with X0/Y0 as the back-right corner instead of correctly setting it as front-left since their firmware incorrectly configured the Ender-5 as homing to X/Y min.

I've fixed that in the configs that we host here, but it doesn't really matter where a machine homes since "home position == X0,Y0" is not universally true.

> Not to start a long drawn out debate, but its just TYPICALLY front left, and for simplicity sake, it would be nice if Marlin would treat every machine the same and when you home the printer it goes to the front left.

> Slicer already has the coordinates in the front left of the software, might as well follow suit with the firmware on the machines!!!

Marlin does this already when you correctly define which direction you home to since Marlin will know when it's at min or max travel limits after homing.

Taking the Ender-5 as an example with `X_HOME_DIR` and `Y_HOME_DIR` set to `1` (max),  `X_BED_SIZE`/`Y_BED_SIZE`/`X_MAX_POS`/`X_MAX_POS` set to `220`, Marlin knows that the home position is X220/Y220 and can move -220mm in X & Y after homing to reach 0,0.

"Home" should not be equated to X0,Y0, but where your endstops are located.